### DISCUSSION TITLE
How to check the performance before and after the script was executed ?

### DISCUSSION THREAD

**Discussion by sanduluca** [Raphire/Win11Debloat#117] [Q&A]
How to check the performance before and after the script was executed ?

Lets say i have a slow computer with a Windows 10. The apps are opening slowly. The PC itself starts slowly. Can I somehow measure the Windows speed before and after ? Some kind of Pagespeed/Lighthouse ?

**Comment by Raphire** [comment]
Heya,

I'm not sure what the best way to measure this would be. The performance improvements would probably be incredibly small on all but the slowest systems. Most of the gains would be from eliminating certain apps and processes from running in the background.

I suppose it's possible to get an idle baseline CPU/RAM usage in Task Manager before and after running the script, but I don't now how reproducible and accurate that would be. To get the most accurate results you'd need to restart your PC, open taskmanager and let it sit for a few minutes to get a baseline number. Background tasks and updates could mess up your numbers here though.

Running CPU benchmarks like cinebench might work, but I doubt the before/after results would be significantly different on any modern system.

Edit: I saw a small increase in cinebench R23 running in a VM. I took the average after a 10 min cinebench run, where I got 4587 points before running the script and 4691 after running the script with the default settings.
